DESCRIPTION

```diff
@@ -0,0 +1,47 @@
+Package: admiralonco
+Type: Package
+Title: Oncology extension package for ADaM in R Asset Library 
+Version: 0.0.0
+Authors@R: c(
+    person("Stefan", "Bundfuss", email = "stefan.bundfuss@roche.com", role = c("aut", "cre")),
+    person("Ross", "Farrugia", role = "aut"),
+    person("Amit", "Jain", role = "aut"),
+    person("Liming (Catherine)", "Jin", role = "aut"),
+    person("F. Hoffmann-La Roche AG", role = c("cph", "fnd")),
+    person("GlaxoSmithKline LLC", role = c("cph", "fnd")),
+    person("Amgen Inc.", role = c("cph", "fnd"))
+    )
+Description: A toolbox for programming oncology specific Clinical Data Standards Interchange Consortium
```

Comment:
Should we also add mention of this being a package extension of {admiral}? - probably obvious by the name but can't harm

Reply:
Agree and added